### PR TITLE
add twitter meta tags

### PR DESCRIPTION
**Pull request by heyitsmaimai**

### CODE REVIEW

**Comment by DanielBrierton** [comment]
Deploy preview for *cdf-website* ready!

Built with commit 4817b6a7586e479beaf6179a23ecd81036ca69a3

https://deploy-preview-15--cdf-website.netlify.com

File: pages/volunteer.vue [hunk 1]
```diff
@@ -25,7 +25,13 @@ export default {
...
         { hid: 'og:image', name: 'og:image', property: 'og:image', content: '' },
-        { hid: 'og:description', name: 'og:description', property: 'og:description', content: 'Dojos benefit from a mix of Mentors providing support, guidance and encouragement to CoderDojo attendees (Ninjas). Maybe you love working with kids, perhaps you want to share your technical skills, or maybe you are creative and have an eye for design. All of these traits make you the perfect Dojo volunteer!' }
+        { hid: 'og:description', name: 'og:description', property: 'og:description', content: 'Dojos benefit from a mix of Mentors providing support, guidance and encouragement to CoderDojo attendees (Ninjas). Maybe you love working with kids, perhaps you want to share your technical skills, or maybe you are creative and have an eye for design. All of these traits make you the perfect Dojo volunteer!' },
+        { hid: 'twitter:title', name: 'twitter:title', property: 'twitter:title', content: 'How you can volunteer with CoderDojo' },
+        { hid: 'twitter:url', name: 'twitter:url', property: 'twitter:url', content: 'https://coderdojo.com' },
```
Comment (Wardormeur) [inline pages/volunteer.vue]: this should be /volunteer

File: pages/start-dojo.vue [hunk 2]
```diff
@@ -22,7 +22,12 @@ export default {
...
         { hid: 'og:type', name: 'og:type', property: 'og:type', content: 'website' },
         { hid: 'og:image', name: 'og:image', property: 'og:image', content: '' },
-        { hid: 'og:description', name: 'og:description', property: 'og:description', content: 'Starting a Dojo is a fun and incredibly rewarding experience. The most important attribute you should have is that you can bring people together for a shared goal. Our registration process will guide you every step of the way making sure you have the most useful resources to hand.' }
+        { hid: 'og:description', name: 'og:description', property: 'og:description', content: 'Starting a Dojo is a fun and incredibly rewarding experience. The most important attribute you should have is that you can bring people together for a shared goal. Our registration process will guide you every step of the way making sure you have the most useful resources to hand.' },
+        { hid: 'twitter:title', name: 'twitter:title', property: 'twitter:title', content: 'Start a CoderDojo club in your local area' },
+        { hid: 'twitter:url', name: 'twitter:url', property: 'twitter:url', content: 'https://coderdojo.com' },
```
Comment (Wardormeur) [inline pages/start-dojo.vue]: this should be /start-a-dojo

File: pages/resources.vue [hunk 3]
```diff
@@ -23,7 +23,12 @@
...
           { hid: 'og:type', name: 'og:type', property: 'og:type', content: 'website' },
           { hid: 'og:image', name: 'og:image', property: 'og:image', content: '' },
-          { hid: 'og:description', name: 'og:description', property: 'og:description', content: 'Our collection of learning and training resources will help you in setting up and running your Dojo. The collection includes educational material, exercises and project ideas to give to the Ninjas at your Dojo as well as links to training materials and content for organisers.' }
+          { hid: 'og:description', name: 'og:description', property: 'og:description', content: 'Our collection of learning and training resources will help you in setting up and running your Dojo. The collection includes educational material, exercises and project ideas to give to the Ninjas at your Dojo as well as links to training materials and content for organisers.' },
+          { hid: 'twitter:title', name: 'twitter:title', property: 'twitter:title', content: 'Our range of learning and training resources' },
+          { hid: 'twitter:url', name: 'twitter:url', property: 'twitter:url', content: 'https://coderdojo.com' },
```
Comment (Wardormeur) [inline pages/resources.vue]: This should be /resources

**Pull request merged**